src/Engine/ProtoCore/DSASM/Executive.cs

```diff
@@ -400,12 +400,12 @@ private void RestoreExecutive(int exeblock)
             pc = (int)rmem.GetAtRelative(StackFrame.FrameIndexReturnAddress).IntegerValue;
         }
 
-        private void PushInterpreterProps(InterpreterProperties properties)
+        public void PushInterpreterProps(InterpreterProperties properties)
```

Comment:
Long story short.

Normally before executing some code / calling a function, the VM internally will call `PushInterpreterProps` to save current executing graph node together with other information, and when returning from a function (executing instruction `RETURN`) or finishing executing code (executing instruction `RETB`), the VM will call `PopInterpreterPros` to pop context information.

But in live execution, the execution is split into two phases: executing new code and applying update. For the later no `RETB` instruction will be executed, we have to manually do cleanup. 

This change has nothing to do with this defect fixing, but just for clean-up purpose.